airflow/models/taskinstancekey.py

```diff
@@ -37,6 +37,7 @@ def primary(self) -> tuple[str, str, str, int]:
     @property
     def reduced(self) -> TaskInstanceKey:
         """Remake the key by subtracting 1 from try number to match in memory information."""
+        # todo (dstandish): remove this property
```

Comment:
todo reminder :)

Reply:
deferred 
![Uploading image.png…]()

Reply:
I created a PR to remove this: https://github.com/apache/airflow/pull/43532 .